_apps/space.md

```diff
@@ -0,0 +1,26 @@
+---
+appid: com.aletheiaware.space
+name: S P A C E
+home: https://github.com/AletheiaWareLLC/spacefynego/
+icon: https://github.com/AletheiaWareLLC/spacefynego/blob/master/ui/data/logo.png?raw=true
+screenshot1: https://space.aletheiaware.com/space.svg
+screenshot2: https://space.aletheiaware.com/space-illustration.svg
+screenshot3: https://space.aletheiaware.com/space-phone.png
+screenshot4: https://space.aletheiaware.com/space-table.png
+
+date:      2021-01-28 12:00:00
+excerpt:   Secure, Private, Storage.
+category:  utility
+developer: Aletheia Ware LLC
+
+git: https://github.com/AletheiaWareLLC/spacefynego.git
+package: aletheiaware.com/spacefynego
```

Comment:
I don't think this is correct, as I could not `fyne get` it, should it have "cmd" on the end?

```suggestion
package: aletheiaware.com/spacefynego/cmd
```

Reply:
Ah quite right, thanks!